src/EditorFeatures/VisualBasic/QuickInfo/SemanticQuickInfoProvider.vb

```diff
@@ -106,6 +107,10 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.QuickInfo
             Return False
         End Function
 
+        Protected Overrides Function GetCaptureFlowAnalysisNodes(semanticModel As SemanticModel, token As SyntaxToken) As ImmutableArray(Of SyntaxNode)
+            Return ImmutableArray(Of SyntaxNode).Empty
```

Comment:
This is because this is a WIP right?

Reply:
No. Showing captures requires quick info for query syntax. Quick info for query syntax isn't supported for VB currently.

Reply:
That makes it sound like this is a WIP then :)   I would expect that if we did this for C#, we'd do this for VB.  Or is there a reason this doesn't make sense for the VB scenario?

Reply:
It totally makes sense for VB. But it requires that we have a symbol to display in quick info that we can hijack for the capture info. This requires to implement  #23049 for VB. (If we go the route of displaying capture info on the first from clause the features are independent and VB support can be included.)